src/h5web/providers/jupyter/api.ts

```diff
@@ -132,15 +132,6 @@ export class JupyterApi implements ProviderAPI {
       };
     }
 
-    // Consider other as unresolved soft links
-    return {
-      ...baseEntity,
-      kind: EntityKind.Link,
-      rawLink: {
-        class: HDF5LinkClass.Soft,
-        title: path,
-        h5path: path,
-      },
-    };
+    throw new Error('Unknown response');
```

Comment:
JLab is not yet "link-aware", so this should not be reachable.

Reply:
It will be reached for datatypes.

Reply:
I've reverted the fallback, but now I'm just returning an unresolved entity without a link.